js-weather/xml-object.js

```diff
@@ -0,0 +1,55 @@
+const citySelect = document.querySelector("#city1");
```

Comment:
Selectは動詞でつかうことが多い単語ですね。
変数名は名詞としましょう。

```suggestion
// せめて名詞に
const citySelector = document.querySelector("#city1");

// selectorだとちょっとわかりにくいので、HTMLのタグっぽく
// 例をいくつか上げておきます
const inputCityForm = ...  // Formだと、<form>タグ全体ぽいので、いまいち
const inputCityField = ...  // fieldはHTML属性であまり使わないですね
const citySelectBox = ...  // selectBoxを使わなくなると変数とHTML要素が合わなくなる可能性あり
const selectedCityElement = ... // Elementを使えばどんなタグになっても、タグっぽいので変更に強め
```

Reply:
d36573f
変数名を変更しました。